sound/soc/sof/intel/hda-dai.c

```diff
@@ -208,9 +209,9 @@ static int hda_dai_hw_free(struct snd_pcm_substream *substream, struct snd_soc_d
 	return hda_link_dma_cleanup(substream, hext_stream, cpu_dai);
 }
 
-static int hda_dai_hw_params(struct snd_pcm_substream *substream,
-			     struct snd_pcm_hw_params *params,
-			     struct snd_soc_dai *dai)
+static int __maybe_unused hda_dai_hw_params(struct snd_pcm_substream *substream,
+					    struct snd_pcm_hw_params *params,
+					    struct snd_soc_dai *dai)
```

Comment:
Should not this be a scope of another series where the SSP/DMIC/SoundWire is added? But at that point the __maybe_unused can be dropped?

Reply:
Whether this is done now or later, the point is that we first need to make some routines available and later use them. I don't want to make SSP/DMIC patches bigger just for a minor Kconfig 'hole' where we have unused stuff.